src/plugins/controls/public/options_list/components/options_list_popover_action_bar.tsx

```diff
@@ -69,22 +64,37 @@ export const OptionsListPopoverActionBar = ({
               onChange={(event) => updateSearchString(event.target.value)}
               value={searchString.value}
               data-test-subj="optionsList-control-search-input"
-              placeholder={
-                totalCardinality
-                  ? OptionsListStrings.popover.getTotalCardinalityPlaceholder(totalCardinality)
-                  : undefined
-              }
+              placeholder={OptionsListStrings.popover.getSearchPlaceholder()}
               autoFocus={true}
             />
           </EuiFlexItem>
+          {allowExpensiveQueries && (
+            <EuiFlexItem grow={false}>
+              <EuiToolTip
+                content={OptionsListStrings.popover.getCardinalityTooltip(totalCardinality)}
+              >
+                <EuiBadge
+                  tabIndex={0}
+                  data-test-subj="optionsList-cardinality-badge"
+                  className="optionsList__actionBarFirstBadge"
+                >
+                  {totalCardinality.toLocaleString()}
+                </EuiBadge>
+              </EuiToolTip>
+            </EuiFlexItem>
```

Comment:
This is how the new cardinality badge interacts with the old "invalid selections" badge:

![Jan-30-2023 13-19-55](https://user-images.githubusercontent.com/8698078/215586244-8b223d09-92a8-45e1-955b-8ae53ee2fbfc.gif)

@elastic/kibana-docs Would appreciate some feedback on the tooltip text here - specifically, for the new cardinality badge. Does `"There are <x> options available."` make it clear that this badge is displaying the cardinality of the given field?

Reply:
**Update:** After feedback about the cardinality badge causing the search bar to change size as the user types, we have decided to use the following design instead:

![image](https://user-images.githubusercontent.com/8698078/216448430-4ae04541-119b-41bd-8b3a-8c224cdeed1a.png)